tendermint/src/lite/types.rs

```diff
@@ -161,13 +226,230 @@ pub enum Error {
 
     InvalidValidatorSet,
     InvalidNextValidatorSet,
-    InvalidCommitValue, // commit is not for the header we expected
-    InvalidCommitLength,
-
-    // TODO(EB): we need to differentiate when this is from
-    // skipping and when it's from verifying !
-    // https://github.com/interchainio/tendermint-rs/issues/119
-    InsufficientVotingPower, /* TODO(Liamsi): change to same name as spec if this changes
-                              * (curently ErrTooMuchChange) */
+    InvalidCommitValue,
+    // commit is not for the header we expected
+    InvalidCommitSignatures,
```

Comment:
Note this is only used by implementation (ie. expected return in validate()). Maybe it should be distinguished from other errors which `lite` itself will return